ClientSupport/src/main/java/io/deephaven/clientsupport/plotdownsampling/BucketState.java

```diff
@@ -310,22 +314,15 @@ public void validate(final boolean usePrev, final DownsampleChunkContext context
                         values[columnIndex].validate(offset, keyChunk.get(indexInChunk), valueChunks[columnIndex],
                                 indexInChunk, trackNulls ? nulls[columnIndex] : null);
                     } catch (final RuntimeException e) {
-                        System.out.println(rowSet);
                         final String msg =
                                 "Bad data! indexInChunk=" + indexInChunk + ", col=" + columnIndex + ", usePrev="
-                                        + usePrev + ", offset=" + offset + ", rowSet=" + keyChunk.get(indexInChunk);
+                                        + usePrev + ", offset=" + offset + ", indexInChunk=" + keyChunk.get(indexInChunk);
+                        log.error().append(msg).append("rowSet=").append(rowSet).endl();
                         throw new IllegalStateException(msg, e);
                     }
                 }
             }
         }
         Assert.eqTrue(makeRowSet().subsetOf(rowSet), "makeRowSet().subsetOf(rowSet)");
     }
-
-    public void close() {
```

Comment:
Are you sure we shouldn't keep this part? I suppose it's safer...

Reply:
What would we want to use it for?

Reply:
When we get rid of bucket states that we're recomputing?

Reply:
What I'm asking is "what would `BucketState.close()` do, if there is no expectation that long-lived RowSets should be closed"?